### PR TITLE
Add support for 204 in render tests

### PR DESCRIPTION
**Pull request by HarelM**
## Launch Checklist

In continue to the last messages in 
- #2497 

I have added support for returning 204 in render tests.
I haven't found an easy way to do that using the `st` package.

### CODE REVIEW

**Review by sbachinin — APPROVED** [review]

File: test/integration/render/run_render_tests.ts [hunk 1]
```diff
@@ -982,15 +982,56 @@ async function executeRenderTests() {
...
 
-    const browser = await puppeteer.launch({headless: !options.openBrowser, args: ['--enable-webgl', '--no-sandbox',
-        '--disable-web-security']});
+    const browser = await puppeteer.launch({
+        headless: !options.openBrowser, 
+        args: [
+            '--enable-webgl', 
+            '--no-sandbox',
+            '--disable-web-security'
+        ]});
```
Comment (ibesora) [inline test/integration/render/run_render_tests.ts:991]: Can we move these definitions in a function somewhere? It seems error prone to have different ways of configuring puppeteer across the code base

**Review by ibesora — APPROVED** [review]
Added a small nit but feel free to ignore it

File: test/integration/render/run_render_tests.ts (hunk 1 above)
Comment (HarelM) [reply test/integration/render/run_render_tests.ts]: We can probably create something in test utils class, but it's just a bit different from test suite to test suite, so I don't know...

Comment (HarelM) [reply test/integration/render/run_render_tests.ts]: Also this was moved to a different PR as main is currently broken, so I needed to merge this fast.
Feel free to open a PR to improve on this.

**Comment by HarelM** [comment]
I'm still waiting to see what the author of `st` will write before I merge this...

**Comment by xabbu42** [comment]
Looking at this code and the answer from the author of st I may be able to make a shorter fix. I'll try to have pull request in the next days. I'm sorry for causing you extra work @HarelM but I would not have had the right direction without it.

**Comment by HarelM** [comment]
Don't worry about the extra work, you made a good observation I missed when reviewing the tests change. 
If you can configure st to return 204 then we are even 🤪

**Comment by HarelM** [comment]
Solved differently by #5269.

**Pull request closed without merging**